README.md

```diff
@@ -53,6 +53,9 @@ If you find a problem or missing feature, open an issue or a pull requests on ht
 
 To have a higher chance of issue being solved, please attach a log file. To record one, go to `settings wheel -> System settings -> Logging -> Enable Debug Logging` and follow the procedure at http://kodi.wiki/view/Log_file/Easy
 
-## Note for Raspberry Pi users:
+# Note for ARM users #
+## Nvidia Shield / most Android boxes ##
+- _Ambilight mode_ doesn't properly work with some HD codecs (usually x265 >1080p) when "allow hardware acceleration - Mediacodec (Surface)" is enabled.
+## Raspberry Pi's: ##
  - Save the add-on configuration by exiting Kodi before shutting down the Pi completely
- - Ambilight mode doesn't work on a Raspberry Pi due to the way it renders video
+ - _Ambilight mode_ doesn't work on a _Raspberry Pi_ due to the way it renders video
```

Comment:
Most arm devices work fine except when "Mediacodec (Surface)" is enabled (but it seems there's a way to use some advanced option to enable it, see the link I posted earlier).  
"Mediacodec (Surface)" is used for some codecs only, usually HD ones, and turning surface render off, has no real "impact".

RPIs are in a different "category", I only added details for "arm / nvidia shield boxes", RPIs have some limitation due to how they render videos, I don't have much more infos, but it's probably hardware related.  

As a "generic rule", you can think like :  
- arm + android = should work  
- arm + linux = might not work
If you're not aware of those devices (this ain't an ad) Odroid boards sounds interresting and I heard it ran kodi + hue, now the good question is "was the feedback made on Android + odroid or Linux + odroid).  
An Odroid board is my next self-xmas present, they support 4k and x265 hw decoding, they accept eMMC modules (and SDcards but eMMC are faster / last longer) and it seems the community around their boards is getting bigger each days (still not an ad).